Doc/library/datetime.rst

```diff
@@ -1600,9 +1600,14 @@ Instance methods:
       >>> dt.isoformat(timespec='microseconds')
       '2015-01-01T12:30:59.000000'
 
-   .. versionchanged:: 3.6
-      Added the *timespec* parameter.
+   If the optional argument *use_utc_designator* is set to :const:`True` and
+   :meth:`tzname` returns exactly ``'UTC'``, then 'Z' will be given as the
+   offset in the formatted string.
 
+   .. versionchanged:: 3.6
+      Added the *timespec* argument.
+   .. versionchanged:: next
+      Added the *use_utc_designator* argument.
```

Comment:
Previous text was correct I think: function definitions have parameters (hence ParamSpec in inspect), function calls take arguments.